DOCS 5888 - Clarified usage of quotation marks. Removed confusing statement. (#885)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/about-mule-message.adoc

* `attributes.headers.'date'` to select `Sun, 20 Jan 2019 18:54:54 GMT` from the header of an HTTP response.
* `attributes.headers.'content-type'` to select the HTTP content type `application/json`.

> For File metadata the attributes are different. For example, when using the `attributes` variable to display File metadata through a Logger component, the Studio console outputs the following example content:

Reviewer: ```suggestion
For file metadata the attributes are different. For example, when using the `attributes` variable to display file metadata through a Logger component, the Studio console displays the following example content:
```

* `attributes.'fileName'` to return the file name: `myJson.json`.
* `attributes.size` to return the size of the file: `22`.

=== Valid Identifiers for Attribute Names

> Attribute names that follow the rules described in xref:dataweave-language-introduction.adoc#valid-identifiers[Rules for Declaring Valid Identifiers] can be accessed and declared without using any quotation marks.

Reviewer: ```suggestion
You can access and declare attribute names that follow the rules described in xref:dataweave-language-introduction.adoc#valid-identifiers[Rules for Declaring Valid Identifiers] without using any quotation marks.
```
Active voice

To use an attribute name that is not a valid identifier, surround the attribute name with any of the following characters:

* single quotation marks (`'`)
* double quotation marks (`"`)

> * backticks (```)

Reviewer: Init. cap. the first word of bullet-list items.

For example, consider the following DataWeave variable declaration:

[source,json,linenums]
----
var myVar = {

>               id : "1234",

Reviewer 2: id is showing up inside a filled in red box. It is not clear why.

Author: yes, this is because the language used to highlight the text. However, this only shows in text editors and in GitHub, but not in the actual build. Just in case, I'll change it.

              "123 abc !@#" : "some_value"
            }
----

> Because attribute name `123 abc !@#` is not a valid identifier, you need to use quotation marks or backticks to declare it.

Reviewer 2: Small nit - change "you need to use"to "you must use"

> To access the value from attribute `123 abc !@#`, which is `"some_value"`, use quotation marks or backticks. For example:

Reviewer: ```suggestion
To access the value from attribute `123 abc !@#`, which is `"some_value"`, use quotation marks or backticks:
```

[source]
----
myVar.'123 abc !@#'
----

> It is not required to use quotation marks or backticks to declare an attribute or to select a value from an attribute when the attribute name is a valid identifier, but it can be done.

Reviewer 3: "but it can be done" : I believe Kim will ask for active voice where possible, e.g., "but you can use these characters around them" or something similar

Reviewer: Duke's right!
```suggestion
Using quotation marks or backticks to declare an attribute or select a value from an attribute when the attribute name is a valid identifier is optional.
```
